packages/@react-stately/layout/src/TableLayout.ts

```diff
@@ -328,7 +336,9 @@ export class TableLayout<T, O extends TableLayoutProps = TableLayoutProps> exten
 
     return {
       layoutInfo,
-      validRect: rect
+      children: [],
+      validRect: rect,
+      node
```

Comment:
Should we update interface `LayoutNode` to require node and children?

Reply:
I tried that before but there are some that don't have associated nodes (the loader and empty state in RSP).